Library/Homebrew/cask/lib/hbc/utils/quarantine.swift

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/swift
+
+import Foundation
+import CoreServices
```

Comment:
Is the `CoreServices` framework actually used?

Reply:
No, it's a leftover from the full-featured script I used as basis. Will remove.